tools/build_script_generator/cmake/cmake_scripts/configure-stm32-gcc.cmake

```diff
@@ -0,0 +1,139 @@
+INCLUDE(./cmake/report-build-options.cmake)
+
+SET(CMAKE_SYSTEM_NAME Generic)
+SET(CMAKE_SYSTEM_PROCESSOR arm)
+
+# Configure toolchain
+
+IF(NOT TOOLCHAIN_PREFIX)
+     SET(TOOLCHAIN_PREFIX "/usr")
+     MESSAGE(STATUS "No TOOLCHAIN_PREFIX specified, using default: " ${TOOLCHAIN_PREFIX})
+ELSE()
+     FILE(TO_CMAKE_PATH "${TOOLCHAIN_PREFIX}" TOOLCHAIN_PREFIX)
+ENDIF()
```

Comment:
Why not assume that the toolchain is just accessible in the path?

Reply:
Good point. Done.